deluge/ui/gtkui/mainwindow.py

```diff
@@ -339,10 +340,17 @@ def is_on_active_workspace(self):
             bool: True if on active workspace (or wnck module not available), otherwise False.
 
         """
-        if not wnck:
+        if not Wnck:
             return True
-        win = wnck.window_get(self.window.window.xid)
-        active_wksp = win.get_screen().get_active_workspace()
+
+        # TODO: Linux specific code required? Above code fails on Ubuntu 14.04...
+        # Necessary to import this for get_xid() to work, at least when testing on Ubuntu 14.04
+        from gi.repository import Gtk, GdkX11  # NOQA
+        screen = Wnck.Screen.get_default()
+        xid = self.window.get_window().get_xid()
+        win = Wnck.Window.get(xid)
+
+        active_wksp = screen.get_active_workspace()
```

Comment:
Should this not be `active_wksp = win.get_screen().get_active_workspace()`? Would negate the need for line 349.